chore(test-coverage): add vitest coverage configuration

nikomakela opened the pull request:
TH-1409.

The Vitest should make a coverage report for Sonarcloud. Therefore, this commit adds `vitest-sonar-reporter` as a dependency and configures the vitest coverage reporters.

nikomakela committed:
chore(test-coverage): add vitest coverage configuration

TH-1409.

The Vitest should make a coverage report for Sonarcloud. Therefore, this
commit adds `vitest-sonar-reporter` as a dependency and configures the
vitest coverage reporters.

Add sonar report files to gitignore.

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-hobbies-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-events-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**HARRASTUKSET-UI branch is deployed to platta: https://harrastukset-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-hobbies-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**HARRASTUKSET-UI branch is deployed to platta: https://harrastukset-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**EVENTS-GRAPHQL-PROXY branch is deployed to platta: https://events-graphql-proxy-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __failed__ for https://harrastukset-pr814.dev.hel.ninja :crying_cat_face::anger::boom::boom:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-sports-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://liikunta-pr814.dev.hel.ninja :laughing::tada::tada::tada:

nikomakela committed:
test: links no more duplicated

TH-1409 TH-1427.

TH-1427 removed the duplicated links, so it should no longer be required
to double the result count in browser tests.

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-events-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-hobbies-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-events-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**TAPAHTUMAT-UI branch is deployed to platta: https://tapahtumat-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**HARRASTUKSET-UI branch is deployed to platta: https://harrastukset-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://tapahtumat-pr814.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
**VENUE-GRAPHQL-PROXY branch is deployed to platta: https://venue-graphql-proxy-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-sports-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**LIIKUNTA-UI branch is deployed to platta: https://liikunta-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**TAPAHTUMAT-UI branch is deployed to platta: https://tapahtumat-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**HARRASTUKSET-UI branch is deployed to platta: https://harrastukset-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**EVENTS-GRAPHQL-PROXY branch is deployed to platta: https://events-graphql-proxy-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://harrastukset-pr814.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
## TestCafe result is __success__ for https://tapahtumat-pr814.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
## TestCafe result is __success__ for https://harrastukset-pr814.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
## TestCafe result is __success__ for https://liikunta-pr814.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
**VENUE-GRAPHQL-PROXY branch is deployed to platta: https://venue-graphql-proxy-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**EVENTS-GRAPHQL-PROXY branch is deployed to platta: https://events-graphql-proxy-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-events-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-hobbies-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**TAPAHTUMAT-UI branch is deployed to platta: https://tapahtumat-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**EVENTS-GRAPHQL-PROXY branch is deployed to platta: https://events-graphql-proxy-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**VENUE-GRAPHQL-PROXY branch is deployed to platta: https://venue-graphql-proxy-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-sports-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __failed__ for https://harrastukset-pr814.dev.hel.ninja :crying_cat_face::anger::boom::boom:

terovirtanen commented:
## TestCafe result is __failed__ for https://tapahtumat-pr814.dev.hel.ninja :crying_cat_face::anger::boom::boom:

terovirtanen commented:
**LIIKUNTA-UI branch is deployed to platta: https://liikunta-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**TAPAHTUMAT-UI branch is deployed to platta: https://tapahtumat-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**HARRASTUKSET-UI branch is deployed to platta: https://harrastukset-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __success__ for https://liikunta-pr814.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
## TestCafe result is __success__ for https://tapahtumat-pr814.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
## TestCafe result is __success__ for https://harrastukset-pr814.dev.hel.ninja :laughing::tada::tada::tada:

terovirtanen commented:
**EVENTS-GRAPHQL-PROXY branch is deployed to platta: https://events-graphql-proxy-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**VENUE-GRAPHQL-PROXY branch is deployed to platta: https://venue-graphql-proxy-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-hobbies-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**FEDERATION-ROUTER branch is deployed to platta: https://events-graphql-federation-events-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**TAPAHTUMAT-UI branch is deployed to platta: https://tapahtumat-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
**HARRASTUKSET-UI branch is deployed to platta: https://harrastukset-pr814.dev.hel.ninja** :rocket::rocket::rocket:

terovirtanen commented:
## TestCafe result is __failed__ for https://tapahtumat-pr814.dev.hel.ninja :crying_cat_face::anger::boom::boom:
